openslides/core/static/js/core/site.js

```diff
@@ -197,6 +197,29 @@ angular.module('OpenSlidesApp.core.site', [
                 data: {extern: true},
                 onEnter: function($window) {
                     $window.location.href = this.url;
+                },
+                resolve: {
```

Comment:
Why do we have to resolve this stuff here. It is already resolved in the slidesProvider.